package.json

```diff
@@ -1,24 +1,19 @@
 {
   "name": "serverless-finch",
-  "version": "3.0.0",
+  "version": "0.0.0-development",
```

Comment:
What's the reason for 0.0.0 here? Will it automatically check the current NPM semver? Or does this need to be updated?

Reply:
It uses the  latest tag on GitHub + the commits since then to determine the new release version before publishing to npm. The 0.0.0 in package json is recommended so we know it's not used or updated

Reply:
Sounds good, thanks!